syntax = "proto3";

package protos;
option go_package = "./proto";
option java_package = "src.main.benchmark";

import "protos/message.proto";


service Python_C {
  rpc Python_C_1(CommonMessage) returns (CommonMessage) {}
  rpc Python_C_2(CommonMessage) returns (CommonMessage) {}
  rpc Python_C_3(CommonMessage) returns (CommonMessage) {}
}
